tests/db.rs

```diff
@@ -214,7 +214,12 @@ pub fn get_structure(
         .expect("0 results, expected one...")
     ;
 
-    row.get(0)
+    /* get_opt(_) returns an error wrapped into an Option
+       object if the row exists and the structure column is null */
+    match row.get_opt(0) {
+        Some(Ok(data)) => Some(data),
+        Some(Err(_)) | None => None,
```

Comment:
if there's an Err , we should panic, no ?

Reply:
or at least consider the test fail ?

Reply:
That makes sense too. We could panic if there is an error (null column) or if the result is None. Ok ?